There seemingly are bugs in the Power9 kernels.
Workaround this by using the generic kernels on Power9 introduced by
https://github.com/flame/blis/commit/ee9ff988c49f16696679d4c6cd3dcfcac7295be7
See https://github.com/flame/blis/issues/621

diff --git a/build/detect/config/config_detect.c b/build/detect/config/config_detect.c
index 5e29def..5603163 100644
--- a/build/detect/config/config_detect.c
+++ b/build/detect/config/config_detect.c
@@ -70,6 +70,8 @@
 int main( int argc, char** argv )
 {
 	arch_t id = bli_cpuid_query_id();
+ 	if(id == BLIS_ARCH_POWER9)
+ 		id = BLIS_ARCH_GENERIC;
 	char*  s  = bli_arch_string( id );
 
 	printf( "%s\n", s );
